src/components/ContentView/ContentView.tsx

```diff
@@ -18,11 +20,48 @@ export type ContentViewProps = {
  */
 export const ContentView: React.FC<ContentViewProps> = ({ contentData, pageTitle }) => {
     const [content, setContent] = useState<ContentData>({ ...contentData } as ContentData);
-
+    const [activeSection, setActiveSection] = useState({section: "", index: 0, numSections: 0});
+    const sections: string[] = [];
     useEffect(() => {
         if (!equal(content, contentData)) {
             setContent({ ...contentData } as ContentData)
         }
+        if (contentData && contentData[pageString]) {
+
+            
+            contentData[pageString].contentOrder!!.forEach(content => {
```

Comment:
I think we should explicitly just check for null/undefined instead of using `!!` as manual assertion. That way we run into less problems in the future.